src/fideslang/models.py

```diff
@@ -879,12 +871,22 @@ class PrivacyDeclaration(BaseModel):
         default_factory=list,
         description="The categories of personal data that this system shares with third parties.",
     )
-    cookies: Optional[List[Cookies]] = Field(
+    cookies: Optional[Any] = Field(  # type: ignore
```

Comment:
just curious, could we change this to `cookies: None`? not a big deal if that causes a lot of typing issues, but that may be the clearest way to indicate via the typing that this should not have any data